src/centos/scripts/start-marklogic.sh

```diff
@@ -203,9 +260,7 @@ function get_host_id {
 #         localhost: if bootstrap host is the localhost
 ################################################################
 function verify_bootstrap_status {
-    local bootstrap_host=$1
-    local bootstrap_host_id=""
-    local localhost_id=""
+    local bootstrap_host=$1 bootstrap_host_id="" localhost_id=""
```

Comment:
same, local should be in it's own line

Reply:
updated var declaration and assignment for variables